enterprise/views.py

```diff
@@ -528,6 +528,12 @@ def get(self, request):  # pylint: disable=too-many-statements
         """
         Render a form to collect user input about data sharing consent.
         """
+        # We need to override this constant here, because it's calcualted at load time and there's no
```

Comment:
This typo is in all the comments.  @Squirrel18

```suggestion
        # We need to override this constant here, because it's calculated at load time and there's no
```

Reply:
Addressed. @Jacatove